src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property is more commonly included in a global CSS reset, rather than on individual elements. Consider moving it to a global selector like `html` or `*`.